icpc-wf/ansible/group_vars/all/all.yml

```diff
@@ -38,6 +38,7 @@ PHP_MAX_FILE_UPLOADS: 101
 # Git repo URL
 DJ_GIT_HOST: "{{MAIN_ADMIN_IP}}"
 DJ_GIT_REPO: "domjudge@{{DJ_GIT_HOST}}:domjudge"
+DJ_GIT_REPO_UPSTREAM: "https://github.com/domjudge/domjudge.git"
 DJ_GIT_REPO_SCRIPTS: "domjudge@{{DJ_GIT_HOST}}:domjudge-scripts-bare"
```

Comment:
Do we need a similar thing for this repo?

Reply:
I think it makes more sense to set the `DJ_GIT_REPO` variable correctly depending on context, instead of duplicating the `create working copy of the domjudge repo` task below.

Reply:
For the SCRIPTS? That probably relevant if we would have admin machines.,

I think we'll never use it but yes, I'll add it.

Reply:
And I agree with Jaap, just for fixing the script this was the quickest fix.